ios/RNAccountKitViewController.m

```diff
@@ -58,7 +58,7 @@ - (void)loginWithPhone: (RCTPromiseResolveBlock)resolve
         UIViewController<AKFViewController> *viewController = [_accountKit viewControllerForPhoneLoginWithPhoneNumber:prefillPhoneNumber state:inputState];
         [self _prepareLoginViewController:viewController];
         UIViewController *rootViewController = [UIApplication sharedApplication].delegate.window.rootViewController;
-        [rootViewController presentViewController:viewController animated:YES completion:NULL];
+        [rootViewController showViewController:viewController sender:nil];
```

Comment:
It would be great if this can be configurable. I think two options here:

- Adding a new field to the `RNAccountKit.configure` argument:

```js
RNAccountKit.configure({
  ...
  viewControllerMode: 'show'|'present' // for iOS only, 'present' by default
})
```

- Changing the signature of `loginWithPhone` and `loginWithEmail` only if we can keep the method backward compatible (It could be more difficult to implement.

```js
RNAccountKit.loginWithEmail({ 
  viewControllerMode: 'show'|'present' // 'present' by default 
})

RNAccountKit.loginWithPhone({ 
  viewControllerMode: 'show'|'present' // 'present' by default 
})
``

Adding @gaguirre to the discussion

Reply:
Thanks for your guide, I will try then back to you 👍